src/Components/Import/Import.tsx

```diff
@@ -1,32 +1,178 @@
-import React, {useState} from 'react';
+import React, { useState } from "react";
+import ReactPlayer from "react-player";
+import {
+  isImage,
+  isVideo,
+  ErrorModal,
+  showError,
+  trimmedName,
+  Media,
+  AddMedia
+} from "..";
```

Comment:
These should all be listed separately, to make it easier to tell where they're being imported from